src/languages/bash.js

```diff
@@ -69,6 +69,11 @@ export default function(hljs) {
     begin: /'/,
     end: /'/
   };
+  const ESCAPED_APOS = {
+    className: '',
```

Comment:
This key shouldn't be necessary at all if you don't want to assign a scope/class to the match.